beamer/tests/agent/unit/test_challenge.py

```diff
@@ -93,8 +95,10 @@ def test_join_false_claim_challenge_only_when_unfilled(filler):
         assert not claim.transaction_pending
 
 
-@patch("beamer.agent.relayer.run_relayer_for_tx")
+@patch("beamer.agent.chain.run_relayer_for_tx")
 def test_optimism_prove(_mocked_relayer_call):
+    timestamp = int(time.time())
+    _mocked_relayer_call.return_value = str(timestamp)
```

Comment:
Nitpick: we should remove the leading underscore from the name, we are actually using it now.